aiogram/dispatcher/filters/builtin.py

```diff
@@ -757,3 +757,72 @@ def __init__(self, is_media_group: bool):
 
     async def check(self, message: types.Message) -> bool:
         return bool(getattr(message, "media_group_id")) is self.is_media_group
+
+
+class StorageDataFilter(BoundFilter):
+    """
+    Check if all items matches the relevant items in the current storage data.
+    """
+
+    key = 'storage_data'
+    ctx_storage_data = ContextVar('user_storage_data')
+
+    def __init__(self, dispatcher, storage_data: dict):
+        from aiogram import Dispatcher
+
+        self.dispatcher: Dispatcher = dispatcher
+        self.storage_data = storage_data
```

Comment:
Storage type hint set in arguments, but dispatcher type hint set inside the function.
Please make it similar way.